integration/client-cli/client-registration-cli/README.md

```diff
@@ -0,0 +1,201 @@
+Keycloak Client Registration CLI
```

Comment:
There is some content in there that is of use to QE team. Also, people may find at least pointers to documentation useful.

Maybe I can put testing notes into this PR's description.